### PR TITLE
Added content as outside collaborator

### PR DESCRIPTION
**Pull request by shashankaws0304**
Added content as outside collaborator

### CODE REVIEW

**Comment by shashank0304** [comment]
Need to add more content.

**Comment by shashank0304** [comment]
Review this.

File: forked/forked.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Adding content as outside collaborator
```
Comment (shashank0304) [inline forked/forked.txt:1]: Its reviewed

**Review by shashank0304 — COMMENTED** [review]
Review completed.

**Comment by shashankaws0304** [comment]
Thank you for adding my pull request

**Pull request merged**